Source/WarpX.cpp

```diff
@@ -416,7 +416,7 @@ WarpX::ReadParameters ()
                 snapshot_interval_is_specified,
                 "When using back-transformed diagnostics, user should specify either dz_snapshots_lab or dt_snapshots_lab.");
 
-            pp.get("gamma_boost", gamma_boost);
+            pp.get("gamma_boost", gamma_boost); // this is read in ReadBoostedFrameParameters()
```

Comment:
Thank you. forgot to remove this comment. It was written when I was trying to understand where the boosted-frame data is read. 
gamma_boost is parsed using pp.get() in WarpX.cpp 
and it is also queried in WarpXUtil.cpp instead of using the already parsed gamma_boost value. This is not necessarily a problem. Just wanted to make a note to myself to see whether the query is really necessary?